scene/resources/packed_scene.cpp

```diff
@@ -43,7 +44,7 @@
 #include "scene/property_utils.h"
 
 #define PACKED_SCENE_VERSION 2
-
+#define META_POINTER_PROPERTY_BASE "metadata/_editor_prop_ptr_"
```

Comment:
```suggestion
#define META_POINTER_PROPERTY_BASE "metadata/_editor_prop_ptr_"

```
btw this could be `constexpr` instead of macro (same with scene version).

Reply:
Not a fan of constexpr myself until it needs to be used as such.